test/IntegrationTests/test_addcustomer.py

```diff
@@ -1,13 +1,15 @@
 import unittest
 import requests
 import logging
+import os
 
 # Set up logging configuration
 logging.basicConfig(level=logging.INFO)
 logger = logging.getLogger(__name__)
 
 class TestAddCustomerAPI(unittest.TestCase):
-    BASE_URL = "http://localhost:8000"
+    # Get the port from environment variables (default to 8000 if not set)
+    BASE_URL = f"http://localhost:{os.getenv('CHAT_SERVICE_PORT', 8000)}"
```

Comment:
Don't set default value for this environmental variable

Reply:
I removed default values from this code.